cli/src/main.rs

```diff
@@ -314,7 +343,11 @@ fn main() -> Result<()> {
         Cli::Chip(Chip::List) => print_families(io::stdout()).map_err(Into::into),
         Cli::Chip(Chip::Info { name }) => print_chip_info(name, io::stdout()),
         Cli::Benchmark { common, options } => benchmark(common, options),
-    }
+    };
+
+    eprintln!("Log file was written to {:?}", log_path);
```

Comment:
I would prefer to have this in the beginng, so it's also visible when a panic occurs. Could also be included in the normal log output to stdout.

I find it a bit verbose to see this everytime, for each command, when I'm not interested in the log.